tests/classification/test_auc.py

```diff
@@ -32,19 +32,29 @@ def sk_auc(x, y):
     return _sk_auc(x, y)
 
 
+def sk_auc_reorder(x, y):
+    x = x.flatten()
+    y = y.flatten()
+    idx = np.argsort(x, kind='stable')
+    x = x[idx]
+    y = y[idx]
+    return _sk_auc(x, y)
+
+
 Input = namedtuple('Input', ["x", "y"])
 
 _examples = []
 # generate already ordered samples, sorted in both directions
-for i in range(4):
-    x = np.random.randint(0, 5, (NUM_BATCHES * 8))
-    y = np.random.randint(0, 5, (NUM_BATCHES * 8))
-    idx = np.argsort(x, kind='stable')
-    x = x[idx] if i % 2 == 0 else x[idx[::-1]]
-    y = y[idx] if i % 2 == 0 else x[idx[::-1]]
-    x = x.reshape(NUM_BATCHES, 8)
-    y = y.reshape(NUM_BATCHES, 8)
-    _examples.append(Input(x=tensor(x), y=tensor(y)))
+for batch_size in (8, 4049):
```

Comment:
that will be very long can we iterate in some steps - 10 or 100?

Reply:
@Borda this is fine, it is not `range(8, 4049)` but just iteration over the two values.